openedx/core/djangoapps/bookmarks/tests/test_tasks.py

```diff
@@ -142,10 +145,11 @@ def test_calculate_course_xblocks_data(self, course_attr):
                     )
 
     @ddt.data(
-        ('course', 47),
-        ('other_course', 34)
+        ('course', 48),
+        ('other_course', 35)
     )
     @ddt.unpack
+    @skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Different query counts in CMS')
```

Comment:
A strategy I use for debugging that kind of thing is to just drop an exception (or a breakpoint) at the point where the code is writing to the cache, and then running the test to see the stack trace.